zio-http/src/test/scala/zio/http/BodySpec.scala

```diff
@@ -29,10 +29,10 @@ object BodySpec extends ZIOSpecDefault {
           ),
           suite("fromFile")(
             test("failure") {
-              val res: URIO[Any, Either[Throwable, Chunk[Byte]]] =
+              val res =
```

Comment:
@adamgfraser  I have ignored the test for now, but I think the test is not correct.